pallets/schemas/src/lib.rs

```diff
@@ -428,231 +486,11 @@ pub mod pallet {
 		// REMOVED propose_to_create_schema() at call index 2
 		// REMOVED create_schema_via_governance() at call index 3
 		// REMOVED create_schema_v2() at call index 4
-
```

Comment:
Shouldn't we first deprecate these and then remove in the next release?

Reply:
We discussed this a couple of weeks ago and decided that, since these are governance-only actions, it's okay to simply remove. Plus, if we deprecate them, we would have to hard-code them to simply fail, because going forward you can't create a Schema without specifying which Intent it belongs to.

We could do something complicated like auto-create an Intent and an associated name, but there's really no point, since it's a governance action that, to date, has only been used internally by @wesbiggs and some test pipelines.